scripts/module/ghostcars.nut

```diff
@@ -0,0 +1,40 @@
+function onScriptLoad() {
+  GHOSTKEY <- BindKey( true, 0x47, 0, 0 ); // G
+  ghost <- array( 100, false );
+}
+
+function onKeyDown( player, id ) {
+  if (id == GHOSTKEY) {
+    ghost[player.ID] = !ghost[player.ID];
+    local message = "Your vehicle is a ghost: ";
+    if (ghost[player.ID]) {
+      if (player.VehicleSlot == 0) { // Player is driver
+        player.Vehicle.IsGhost = true;
+      }
+      message += "ON";
+    } else {
+      if (player.VehicleSlot == 0) {
+        player.Vehicle.IsGhost = false;
+      }
+      message += "OFF";
+    }
+    MessagePlayer(message, player);
+  }
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  cmd = cmd.tolower();
+  if (cmd == "help") {
+    MessagePlayer("[#8080A0][G] - Ghostcar", player);
+  }
+}
+
+function onPlayerEnterVehicle( player, vehicle, door ) {
+  if (ghost[player.ID] && door == 1) {
+    vehicle.IsGhost = true;
+  }
```

Comment:
Wouldn't this also happen if a passenger entered your car?

Reply:
Quite possible - door `1` is documented as drivers door. Guess wiki means `[front] driver side`

Reply:
This event should trigger when player has entered the vehicle. This means the (newly added) check for `player.VehicleSlot` should be fine.